src/properties.js

```diff
@@ -458,6 +458,55 @@ export function suspendableProp(options) {
     )
 }
 
+export function delayableHandler(options) {
+  /*
+  Delays `[handler]` calls until after `[filter]` become true.
+  */
+  const handler = options.handler
+  const filter = options.filter
```

Comment:
As for other decorator creators, it could benefit from default values.
To follow conventions and explicit parameters, `handlerName` and `sentinelName` are clearer than `handler` and `filter`.